lib/modulr/client.rb

```diff
@@ -70,6 +73,9 @@ def request_options(_method, _path, data, _options)
     end
 
     def add_auth_options!(options)
+      options[:headers][:authorization] = @apikey
```

Comment:
If requested url is sandbox modulr environment we do not need to send signature headers but authorization with apikey value instead.